modules/uhsg_rest/src/Plugin/rest/resource/DegreeProgrammeResource.php

```diff
@@ -147,6 +157,7 @@ private function getNodeCount(TermInterface $term, $content_type) {
         ->condition('field_news_degree_programme', $term->tid->value)
         ->condition('type', $content_type)
         ->condition('langcode', $languageCode)
+        ->accessCheck(false)
```

Comment:
Should we add a status condition to only include published news now that access check won't do it for us?